lib/Optimizer/Transforms/MultiControlDecomposition.cpp

```diff
@@ -0,0 +1,158 @@
+/*************************************************************** -*- C++ -*- ***
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ *******************************************************************************/
```

Comment:
nit: line length 81.

Reply:
I think I will do a PR to change all headers, because we have being going back and forth on this. (I have seen PR changing headers in one way, just that another change it back)